base/Dockerfile

```diff
@@ -221,4 +221,15 @@ WORKDIR "${HOME}"
 # This part of the base image contains SWAN specific customizations
 # and python packages.
 
-RUN pip install --no-cache-dir swanheader==2.0.1
\ No newline at end of file
+RUN pip install --no-cache-dir swanheader==2.0.1
+
+USER root
+
+# This setting disables the creation of the home directory for
+# new users, which is enabled by default. It is needed, so that
+# the start.sh script is able to create the directory and copy
+# the files from jovyan to the new user home.
```

Comment:
Mention that this is the default in Ubuntu, which is what the upstream image uses. So, we're aligning the behaviour because there are assumptions.